src/stylesheets/elements/_inputs.scss

```diff
@@ -82,7 +82,9 @@ textarea {
   height: 16rem;
 }
 
+// padding right added to avoid text overlapping with bg image
 select {
+  padding-right: 3.2rem;
```

Comment:
Although it's already incorrect, could you change this so that every property is in alphabetical order? https://pages.18f.gov/frontend/css-coding-styleguide/format/